### DISCUSSION TITLE
During use, some pictures can reach 120 fps, others can't. The same delay for the same operating system, I don't know what happens

### DISCUSSION THREAD

**Discussion by youhers** [rustdesk/rustdesk#3226] [General]
Basically the PC configuration is the same, but FPS has a lot of differences, including basically the same software installed in it, one PC in my client can basically reach 120 fps, the other clients can not reach 120 fps, which is amazing.

**Comment by 21pages** [comment]
120fps, could you tell your PC configuration?

**Reply by youhers** [reply]
I5-9400  8G  1T   Intel（R）UHD Graphies 630   WindowsServer2008R2 Enterprise

**Reply by youhers** [reply]
Wrong, it can reach 60fps

**Reply by Moonbesk** [reply]
How to configure FPS to achieve it? What is the default FPS of the software?
